ixofoundation/ixo-webclient
# feat: add profile cards

## Scope
https://nonaredteam.atlassian.net/browse/IXO-352

## Work Done
Added Profile card component
Added the section to the Project page

## Steps to Test
Make sure the tests pass
Make sure the styling looks as per the designs
You can add these users to render the component:
```
props.project.users = [
    {
      imageUrl: 'https://pds_pandora.ixo.world/public/smsal67t2pgk8ss96ss',
      name: 'Joyce Montgomery',
      role: 'Co-Founder & CEO',
      linkedinLink:
        'https://ch.linkedin.com/company/ixofoundation?trk=public_profile_topcard_current_company',
      twitterLink: 'https://twitter.com/ixoworld',
    },
    {
      imageUrl: 'https://pds_pandora.ixo.world/public/smsal67t2pgk8ss96ss',
      name: 'Islam Naidoo',
      role: 'CTO',
      linkedinLink:
        'https://ch.linkedin.com/company/ixofoundation?trk=public_profile_topcard_current_company',
      twitterLink: 'https://twitter.com/ixoworld',
    },
    {
      imageUrl: 'https://pds_pandora.ixo.world/public/smsal67t2pgk8ss96ss',
      name: 'Jonathan Mike',
      role: 'Designer',
      linkedinLink:
        'https://ch.linkedin.com/company/ixofoundation?trk=public_profile_topcard_current_company',
      twitterLink: 'https://twitter.com/ixoworld',
    },
    {
      imageUrl: 'https://pds_pandora.ixo.world/public/smsal67t2pgk8ss96ss',
      name: 'Daniel Ingram',
      role: 'Co-Founder & CEO',
      linkedinLink:
        'https://ch.linkedin.com/company/ixofoundation?trk=public_profile_topcard_current_company',
      twitterLink: 'https://twitter.com/ixoworld',
    },
  ]
```

## Screenshots
![image](https://user-images.githubusercontent.com/46312800/79734277-bcd71780-82f6-11ea-85b3-e9715f42b692.png)
